### PR TITLE
Only write busy to \Fcmderr if \Fcmderr is 0.

### PR DESCRIPTION
**Pull request by timsifive**

### CODE REVIEW

File: xml/dm_registers.xml [hunk 1]
```diff
@@ -388,18 +393,24 @@
...
-      <field name="autoexecprogbuf" bits="31:16" access="R/W" reset="0">
-          When a bit in this field is 1, read or write accesses to the corresponding {\tt progbuf} word
-          cause the command in \Rcommand to be executed again.
-      </field>
-      <field name="0" bits ="15:12" access="R" reset = "0"/>
-      <field name="autoexecdata" bits="11:0" access="R/W" reset="0">
-          When a bit in this field is 1, read or write accesses to the corresponding {\tt data} word
-          cause the command in \Rcommand to be executed again.
-      </field>
+        Writing while an abstract command is executing causes \Fcmderr to be
```
Comment (mwachs5) [inline xml/dm_registers.xml]: Writing this register?

File: xml/dm_registers.xml [hunk 2]
```diff
@@ -316,6 +316,9 @@
     <!-- =============== abstract commands =============== -->
 
     <register name="Abstract Control and Status" short="abstractcs" address="0x16">
+        Writing while an abstract command is executing causes \Fcmderr to be
```
Comment (mwachs5) [inline xml/dm_registers.xml]: Writing this register?

File: xml/dm_registers.xml (hunk 1 above)
Comment (timsifive) [reply xml/dm_registers.xml]: Yes. Done.

File: debug_module.tex [hunk 3]
```diff
@@ -181,6 +181,18 @@ \section{Abstract Commands} \label{abstractcommands}
 depends on the command being executed, and is MXLEN where not explicitly
 specified.
 
+\begin{commentary}
+    The Abstract Command interface is designed to allow a debugger to write
+    commands as fast as possible, and then later check whether they worked.  In
```
Comment (mwachs5) [inline debug_module.tex]: Worked-> completed without error

File: debug_module.tex [hunk 4]
```diff
@@ -181,6 +181,18 @@ \section{Abstract Commands} \label{abstractcommands}
...
+    the common case the debugger will be much slower than the target and
+    commands don't fail, which allows for maximum throughput. If there is a
```
Comment (mwachs5) [inline debug_module.tex]: Don’t fail-> succeed

**Comment by timsifive** [comment]
Thanks!

**Pull request merged**